docs/recurring-events-plan.md

```diff
@@ -0,0 +1,71 @@
+### Recurring Events Plan
+
+We are using these two articles as our reference point:
+https://github.com/bmoeskau/Extensible/blob/master/recurrence-overview.md
+https://vertabelo.com/blog/again-and-again-managing-recurring-events-in-a-data-model/
+
+#### Schema
+
+CalendarEntry:
+eventId
+creatorId
+title
+description
+allDay
+startTimeUtc
+endTimeUtc
+recurrenceEndsUtc
+recurring
+recurrencePattern
```

Comment:
duration missing?

Reply:
we no longer need duration with the new way of doing things